lambda/README.md

```diff
@@ -1,14 +1,21 @@
 # Running a lambda on localstack
 
-1. Create a zip
+## Dependencies
+
+1. [Rialto-api / localstack](https://github.com/sul-dlss-labs/rialto-api/)
```

Comment:
I don't think that this project depends on rialto-api.  They are two separate components and they do not have any dependency, right?

Reply:
The dependencies we *do* have is SNS and ES